test.py

```diff
@@ -0,0 +1,5 @@
+from aiconfig import AIConfigRuntime
```

Comment:
Was this file meant to be committed? Looks like a local test

Reply:
Not intended to be committed, thanks for catching. Removed.